src/main/resources/db/liq/changelog.xml

```diff
@@ -0,0 +1,163 @@
+            </column>
+            <column name="category" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="language_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <!--> <-->
+    <changeSet  author="aleksdenni"  id="4">
+        <addForeignKeyConstraint  baseColumnNames="language_id"
+                                  baseTableName="categories"
+                                  constraintName="fk_categories_languages"
+                                  referencedColumnNames="language_id"
+                                  referencedTableName="languages"/>
+    </changeSet>
+    <!--> <-->
+    <changeSet id="5" author="aleksdenni">
+        <createTable tableName="questions">
+            <column name="question_id" type="int" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="question" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="category_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <!--> <-->
+    <changeSet  author="aleksdenni"  id="6">
+        <addForeignKeyConstraint  baseColumnNames="category_id"
+                                  baseTableName="questions"
+                                  constraintName="fk_questions_categories"
+                                  referencedColumnNames="category_id"
+                                  referencedTableName="categories"/>
+    </changeSet>
+    <!--> answer ? structure <-->
+    <changeSet id="7" author="aleksdenni">
+        <createTable tableName="options">
+            <column name="option_id" type="int" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="answer" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="question_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="correct" type="bool">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <!--> <-->
+    <changeSet  author="aleksdenni"  id="8">
+        <addForeignKeyConstraint  baseColumnNames="question_id"
+                                  baseTableName="options"
+                                  constraintName="fk_options_questions"
+                                  referencedColumnNames="question_id"
+                                  referencedTableName="questions"/>
+    </changeSet>
+    <!--> successful? name column? <-->
+    <changeSet id="9" author="aleksdenni">
+        <createTable tableName="passed_test">
```

Comment:
@NatMyronenko do you agree on table name? Is it descriptive enough?

Reply:
table name -> (change to) user_passed_test

Reply:
@NatMyronenko do you permit the name of the "successful" column?

Reply:
No, tableName "passed_test"  change to "user_passed_test", this name has a more detailed description and also corresponds to the name in the relations hierarchy created by our mentor.

Reply:
@aleksdenni No, tableName "passed_test" change to "user_passed_test", this name has a more detailed description and also corresponds to the name in the relations hierarchy created by our mentor.